### PR TITLE
fix: always get version tags

### PR DESCRIPTION
**Pull request by xynydev**
* previous configuration did not set major version tags correctly when
recipe said "latest"

### CODE REVIEW

**Comment by xynydev** [comment]
My additions should now work correctly, but there's some other tag-related error I can't really find the source of. Someone else take a look?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -96,13 +102,7 @@ jobs:
...
               echo "${TAG}"
           done
-          echo "alias_tags=${alias_tags[*]}" >> $GITHUB_OUTPUT
-
-      - name: Get current version
-        id: labels
-        run: |
-          ver=$(skopeo inspect docker://${{ env.BASE_IMAGE_URL }}:${{ env.FEDORA_MAJOR_VERSION }} | jq -r '.Labels["org.opencontainers.image.version"]')
-          echo "VERSION=$ver" >> $GITHUB_OUTPUT
+          echo "alias_tags=${alias_tags[*]}" >> $GITHU:B_OUTPUT
```
Comment (scottames) [inline .github/workflows/build.yml]:
```suggestion
          echo "alias_tags=${alias_tags[*]}" >> $GITHUB_OUTPUT
```
this may be your issue?

Comment (xynydev) [reply .github/workflows/build.yml]: thanks, totally missed that since that wasn't an intentional change and the GH diff didn't really show it that well

Comment (xynydev) [reply .github/workflows/build.yml]: yeah, that was it

**Review by EyeCantCU — APPROVED** [review]

**Pull request merged**